RFC.md

```diff
@@ -147,49 +147,84 @@ Try out the new types here: TODO Playground Link
 
 # `Auth` Changes
 
-- CognitoUser object is no longer required
+Amplify is proposing the following changes for the `Auth` category.
```

Comment:
```suggestion
Amplify is proposing the following changes for the `Auth` category. Similar changes will be applied across all of the `Auth` APIs but examples for specific APIs are highlighted below.
```

Added some wording to make it clear these changes will be applied across auth (which customers can experiment with in the playground)